packages/itwinui-react/src/core/ThemeProvider/ThemeContext.tsx

```diff
@@ -9,7 +9,7 @@ export const ThemeContext = React.createContext<
   | {
       theme?: ThemeType;
       themeOptions?: ThemeOptions;
-      portalContainerRef?: React.RefObject<HTMLElement>;
```

Comment:
Just wondering, why was `portalContainer` needed when `portalContainerRef` already exists? I guess something along the lines of when the `portalContainerRef` that was passed was changed by the user's code, the `portalContainerRef` was not updated inside our code?

Reply:
I meant to ask that why was `portalContainerRef` replaced by `portalContainer` ([diff](https://github.com/iTwin/iTwinUI/pull/1477/files#diff-653d15c515af2a52d6613822762b8a6f494de559d9c9b9c0f7833d66f35e972dL12-R12))? Like what were the problems with `portalContainerRef` that prompted this change?

Reply:
did you see where it's used? it's no longer a ref

Reply:
iiuc, the main change is that instead of using a ref (`portalContainerRef`), you are now using the actual Element (`portalContainer`). So I am wonder, can the same result behavior of this PR be achieved by reusing `portalContainerRef`?

Reply:
Sounds good, then the current implementation using `useState` works well.